centreon/src/Core/Security/Vault/Infrastructure/API/CreateVaultConfiguration/CreateVaultConfigurationController.php

```diff
@@ -21,7 +21,7 @@
 
 declare(strict_types=1);
 
-namespace Core\Security\Vault\Infrastructure\Api\CreateVaultConfiguration;
+namespace Core\Security\Vault\Infrastructure\API\CreateVaultConfiguration;
```

Comment:
🚨 A t the moment, we have **`API`** and **`Api`** namespaces (the second seems to be more used)
👉️ We need to fix a "norm" for that.

Reply:
The change of namespace was proposed by Laurent as it is rather an abbreviation than a name. To be discussed